core/driver/sources/src/main/resources/reference.conf

```diff
@@ -52,18 +62,108 @@ probe {
 
   // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
   resolvers {
+
     intellij {
-      repositories = []
-    }
-    // default configuration for `plugins` is Plugins(None) - adding possible parameters in comments for reference
-    plugins {
-      // `repository.uri` is the URI of the plugin repository - use this config if you want to use a non-standard
-      // plugin repository. The standard one used by ide-probe is: https://plugins.jetbrains.com/plugin/download
-      # repository.uri = ""
-    }
-    jbr {
-      repositories = []
+
+      // `intellij.artifact` contains the name of the artifact that should be used as the IntelliJ IDEA instance.
+      // For now, there are two possible values:
+      // "ideaIC" - for the Community edition and "ideaIU" - for the Ultimate Edition
+      artifact = "ideaIC"  # or "ideaIU" - for the Ultimate Edition
```

Comment:
There are ofc other JetBrains products with their respective artifact names... but actually, does ide-probe support anything beyond IntelliJ currently? 🤔

Reply:
Only IntelliJ IDEA is supported for now. Maybe move from `probe.resolvers.intellij.artifact` to `probe.resolvers.artifact` to support possibility of adding other JetBrains products??

Reply:
Huh such moving (renaming) of the key would be a breaking change right? 🤔 I'd leave as is, to be re-considered once the need for support of other IDEs is really needed

Reply:
E.g. as for git-machete-intellij-plugin, I haven't got any reports that the plugin, once thoroughly tested on IntelliJ, ever fails on other IDEs

Reply:
Ok, but this config: `probe.resolvers.intellij.artifact` is a new one anyway. I'm adding it in this PR... So no breaking current code from master branch.

Reply:
@lukaszwawrzyk are you aware of any corner case where a plugin would NOT work on the non-IntelliJ IDEs even if it works on IntelliJ (ofc assuming all dependencies from plugin.xml are satisfied by that IDE, but that's enforced statically anyway &mdash; you won't even install the plugin if such dependencies are missing).

Reply:
> Ok, but this config: probe.resolvers.intellij.artifact is a new one anyway. I'm adding it in this PR... So no breaking of current code from master branch.

Aaa okay, so indeed, let's make it future-proof by avoiding explicit references to `intellij`

Reply:
Ok, moved from `probe.resolvers.intellij.artifact` to `probe.resolvers.artifact` as discussed

Reply:
@PawelLipski I recall for example that PyCharm does not behave or look the same in all situations as IntelliJ with Python plugin.